.gitignore

```diff
@@ -23,3 +23,4 @@ vendor/pkg/
 go-test-report.json
 profile.out
 junit-report.xml
+.idea/
```

Comment:
Can you remove this change? It is not related to the fix, and we don't add editor-specific ignores here (see https://github.com/moby/moby/blob/d4306e4b331b49f086b9bec22421879c3e58072d/.gitignore#L1-L3)